maintenance/scripts/channel_minmax_to_table.py

```diff
@@ -94,10 +92,10 @@ def run(username, password, plate_id, host, port):
 
         print("Adding data: ", len(data))
         table.addData(data)
-        table.close()
 
         print("table closed...")
         orig_file = table.getOriginalFile()
+        table.close()
```

Comment:
so the table could now never be closed if an error is thrown during the save step below

Reply:
It was also possible before that it didn't get closed if an exception was thrown between table creation and closing.
But yes, a try/finally would be safer. I'll add that...

Reply:
Fixed in 9632c0d